---a/net/minecraft/event/ClickEvent.java
+++b/net/minecraft/event/ClickEvent.java
@@ -1,80 +1,62 @@
 package net.minecraft.event;
 
-import com.google.common.collect.Maps;
 import java.util.Map;
 
-public class ClickEvent
-{
+import com.google.common.collect.Maps;
+
+public class ClickEvent {
+
     private final ClickEvent.Action action;
     private final String value;
     private static final String __OBFID = "CL_00001260";
 
-    public ClickEvent(ClickEvent.Action theAction, String theValue)
-    {
+    public ClickEvent(ClickEvent.Action theAction, String theValue) {
         this.action = theAction;
         this.value = theValue;
     }
 
-    public ClickEvent.Action getAction()
-    {
+    public ClickEvent.Action getAction() {
         return this.action;
     }
 
-    public String getValue()
-    {
+    public String getValue() {
         return this.value;
     }
 
-    public boolean equals(Object p_equals_1_)
-    {
-        if (this == p_equals_1_)
-        {
+    public boolean equals(Object p_equals_1_) {
+        if (this == p_equals_1_) {
             return true;
-        }
-        else if (p_equals_1_ != null && this.getClass() == p_equals_1_.getClass())
-        {
-            ClickEvent clickevent = (ClickEvent)p_equals_1_;
+        } else if (p_equals_1_ != null && this.getClass() == p_equals_1_.getClass()) {
+            ClickEvent clickevent = (ClickEvent) p_equals_1_;
 
-            if (this.action != clickevent.action)
-            {
+            if (this.action != clickevent.action) {
                 return false;
-            }
-            else
-            {
-                if (this.value != null)
-                {
-                    if (!this.value.equals(clickevent.value))
-                    {
+            } else {
+                if (this.value != null) {
+                    if (!this.value.equals(clickevent.value)) {
                         return false;
                     }
-                }
-                else if (clickevent.value != null)
-                {
+                } else if (clickevent.value != null) {
                     return false;
                 }
 
                 return true;
             }
-        }
-        else
-        {
+        } else {
             return false;
         }
     }
 
-    public String toString()
-    {
+    public String toString() {
         return "ClickEvent{action=" + this.action + ", value=\'" + this.value + '\'' + '}';
     }
 
-    public int hashCode()
-    {
+    public int hashCode() {
         int i = this.action.hashCode();
         i = 31 * i + (this.value != null ? this.value.hashCode() : 0);
         return i;
     }
 
-    public static enum Action
-    {
+    public static enum Action {
         OPEN_URL("open_url", true),
         OPEN_FILE("open_file", false),
         RUN_COMMAND("run_command", true),
@@ -88,38 +88,31 @@
 
         private static final String __OBFID = "CL_00001261";
 
-        private Action(String p_i45155_3_, boolean p_i45155_4_)
-        {
+        private Action(String p_i45155_3_, boolean p_i45155_4_) {
             this.canonicalName = p_i45155_3_;
             this.allowedInChat = p_i45155_4_;
         }
 
-        public boolean shouldAllowInChat()
-        {
+        public boolean shouldAllowInChat() {
             return this.allowedInChat;
         }
 
-        public String getCanonicalName()
-        {
+        public String getCanonicalName() {
             return this.canonicalName;
         }
 
-        public static ClickEvent.Action getValueByCanonicalName(String p_150672_0_)
-        {
-            return (ClickEvent.Action)nameMapping.get(p_150672_0_);
+        public static ClickEvent.Action getValueByCanonicalName(String p_150672_0_) {
+            return (ClickEvent.Action) nameMapping.get(p_150672_0_);
         }
 
-        static
-        {
+        static {
             ClickEvent.Action[] var0 = values();
             int var1 = var0.length;
 
-            for (int var2 = 0; var2 < var1; ++var2)
-            {
+            for (int var2 = 0; var2 < var1; ++var2) {
                 ClickEvent.Action var3 = var0[var2];
                 nameMapping.put(var3.getCanonicalName(), var3);
             }
         }
     }
-}
+}
